packages/navigation/package.json

```diff
@@ -27,12 +27,12 @@
     "lint": "eslint \"src/**/*.{js,ts,tsx}\""
   },
   "devDependencies": {
-    "@react-navigation/core": "^6.0.0",
+    "@react-navigation/core": "^7.7.0",
     "react-native-turbo": "*"
   },
   "peerDependencies": {
-    "@react-navigation/core": ">=6.0.0",
-    "@react-navigation/native": ">=6.0.0",
+    "@react-navigation/core": ">=7.7.0",
+    "@react-navigation/native": ">=7.0.19",
```

Comment:
If `react-navigation` v6 is still supported via the custom export, should the dependency requirement here be loosened again?

Reply:
ahh hood point, restored it in [7cfeedf](https://github.com/software-mansion-labs/react-native-turbo-demo/pull/217/commits/7cfeedfc06aa1f5104d6c89eb0d955967b2ec09c)